### DISCUSSION TITLE
Error not work RN

### DISCUSSION THREAD

**Discussion by Victorcassiano** [software-mansion/react-native-reanimated#2346] [Q&A]
I did the step by step to install Moti, using only React Native, without Expo.
returned these errors to me:

<img src="https://i.imgur.com/a551x0W.png" alt="Minha Figura" width="400px">

<img src="https://i.imgur.com/TbOZ8yd.png" alt="Minha Figura" width="900px">

I've remade the step by step several times

**Comment by piaskowyk** [comment]
You need to update reanimated version, we got rid of getEnforcing(), or you missed some configuration steps - https://docs.swmansion.com/react-native-reanimated/docs/installation

**Reply by piaskowyk** [reply]
Especially adding the babel plugin.
